pvsite_datamodel/read/latest_forecast_values.py

```diff
@@ -22,6 +23,7 @@ def get_latest_forecast_values_by_site(
     day_ahead_hours: Optional[int] = None,
     day_ahead_timezone_delta_hours: Optional[float] = 0,
     model_name: Optional[str] = None,
+    apply_curtailments: bool = True,
```

Comment:
please remove this, and below in this file. 

We are likely to apply curtailments before the forecast is saved